app/src/test/java/camp/nextstep/edu/github/GithubRepositoriesViewModelTest.kt

```diff
@@ -0,0 +1,50 @@
+package camp.nextstep.edu.github
+
+import camp.nextstep.edu.github.domain.GetRepositoriesUseCase
+import camp.nextstep.edu.github.domain.GithubRepo
+import camp.nextstep.edu.github.domain.GithubRepositories
+import com.google.common.truth.Truth.assertThat
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.DelicateCoroutinesApi
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+@DelicateCoroutinesApi
+class GithubRepositoriesViewModelTest {
+    private lateinit var viewModel: GithubRepositoriesViewModel
+    private lateinit var useCase: GetRepositoriesUseCase
+
+    @Before
+    fun setUp() {
+        useCase = mockk(relaxed = true)
+        viewModel = GithubRepositoriesViewModel(useCase)
+        Dispatchers.setMain(Dispatchers.Unconfined)
+    }
+
+    @After
+    fun tearDown() {
+        Dispatchers.resetMain()
+    }
+
+    @Test
+    fun `Github Repo 목록 로딩 이벤트가 발생하면 목록을 가져올 수 있어야 한다`() {
+        //given
+        coEvery { useCase.invoke() } returns flow {
```

Comment:
```suggestion
        every { useCase.invoke() } returns flow {
```
flow는 suspend가 아니므로, every로 하시는 것이 좋을 것 같아요.